packages/charts/src/charts/bubble/bubble-chart.component.tsx

```diff
@@ -117,6 +127,9 @@ export const BubbleChart: FC<Props> = ({
         yScaleSettings={yScaleSettings}
       >
         <Grid xScale={xScale} yScale={yScale} />
+        {(xAxisTitle || yAxisTitle) && (
+          <AxisTitle x={xAxisTitle} y={yAxisTitle} {...axisTitleSettings} />
```

Comment:
Some of charts support `vertical` and `horizontal` layout which replace the axes position. This use case is not supported in that pull request.